conformance/utils/config/timeout.go

```diff
@@ -19,6 +19,10 @@ package config
 import "time"
 
 type TimeoutConfig struct {
+	// CleanupComplete represents the time block between test cases waiting for resource cleanup to complete.
+	// Max value for conformant implementation: None
+	CleanupComplete time.Duration
```

Comment:
Maybe we should call this `TestIsolation` so it's clear that we're trying to isolate tests from each other. I think CleanupComplete may not be as obvious that it runs between every test.

Reply:
Done.